Backend/suitable_lobby.py

```diff
@@ -0,0 +1,125 @@
+from itertools import combinations
+
```

Comment:
the method order needs to be discussed since it doesn't match the rest of the restylized backend.

Reply:
This has been discussed- we will keep this order convention and change the rest of the backend.